lib/features/destination_picker/presentation/destination_picker_controller.dart

```diff
@@ -88,9 +88,9 @@ class DestinationPickerController extends BaseMailboxController {
     this._searchMailboxInteractor,
     this._createNewMailboxInteractor,
     this._refreshAllMailboxInteractor,
-    this._verifyNameInteractor,
-    treeBuilder,
-  ) : super(treeBuilder);
+    TreeBuilder treeBuilder,
+    VerifyNameInteractor verifyNameInteractor,
```

Comment:
I remember, we did not support `create folder` in destination picker. Why we need this `VerifyNameInteractor` here?

Reply:
Oh, because you didn't do this project or your memory is so bad :)). We have `create folder` support in `destination picker`. Please re-read the code for me.